pep/app/pods/components/modal-dialogs/user/about/template.hbs

```diff
@@ -0,0 +1,24 @@
+<ModalDialog @onClose={{@onClose}} @size="md" as |Modal|>
+    <Modal.header @title={{t "about.title"}} />
+    <Modal.body>
+        <div class="mb-2">
+            <div>{{t "about.client"}}</div>
+            <div class="text-muted small">{{@options.clientBuildVersion}}</div>
+        </div>
+        <div class="mb-2">
+            <div>{{t "about.server"}}</div>
+            <div class="text-muted small">{{@options.serverInformation.text_server_version}}</div>
+            <div class="text-muted small">{{@options.serverInformation.dataSource}}</div>
+        </div>
```

Comment:
in my stickler effort to strive for more semantic markup in this project 😛 (since accessibility is a big element), content like this should probably be structured more like:
```hbs
<h5 class="mb-0">{{t "about.server"}}</h5>
<p class="text-muted small">
    {{@options.serverInformation.text_server_version}}<br>
    {{@options.serverInformation.dataSource}}
</p>
```